client/app/requests/order-explorer/order-explorer.component.js

```diff
@@ -243,6 +243,7 @@ function ComponentController($filter, lodash, ListView, Language, OrdersState, E
     if (!refresh) {
       vm.loading = true;
     }
+    getFilterCount();
```

Comment:
thinking ya might want to throw this in the handleSuccess, otherwise we'll be pounding the user with error messages (in the case it fails)